*** orig/isolinux.cfg	2016-10-12 04:45:19.000000000 +0200
--- custom/isolinux.cfg	2017-02-12 00:36:09.201190417 +0100
***************
*** 4,7 ****
  include menu.cfg
  default vesamenu.c32
  prompt 0
! timeout 0
--- 4,7 ----
  include menu.cfg
  default vesamenu.c32
  prompt 0
! timeout 100
*** orig/boot/grub/grub.cfg	2018-04-25 22:11:06.000000000 +0200
--- custom/boot/grub/grub.cfg	2018-06-01 22:47:30.953007353 +0200
***************
*** 9,14 ****
--- 9,15 ----
  
  set menu_color_normal=white/black
  set menu_color_highlight=black/light-gray
+ set timeout=10
  
  menuentry "Install" {
  	set gfxpayload=keep
