diff -cr orig/boot/grub/grub.cfg custom/boot/grub/grub.cfg
*** orig/boot/grub/grub.cfg	2016-10-12 04:45:19.000000000 +0200
--- custom/boot/grub/grub.cfg	2017-02-12 00:34:01.977187336 +0100
***************
*** 10,15 ****
--- 10,21 ----
  set menu_color_normal=white/black
  set menu_color_highlight=black/light-gray
  
+ menuentry "Unattended Install" {
+ 	set gfxpayload=keep
+ 	linux	/linux auto=true priority=critical preseed/file=/preseed.cfg --- quiet
+ 	initrd	/initrd.gz
+ }
+ 
  menuentry "Install" {
  	set gfxpayload=keep
  	linux	/linux --- quiet
diff -cr orig/isolinux.cfg custom/isolinux.cfg
*** orig/isolinux.cfg	2016-10-12 04:45:19.000000000 +0200
--- custom/isolinux.cfg	2017-02-12 00:36:09.201190417 +0100
***************
*** 4,7 ****
  include menu.cfg
  default vesamenu.c32
  prompt 0
! timeout 0
--- 4,7 ----
  include menu.cfg
  default vesamenu.c32
  prompt 0
! timeout 100
diff -cr orig/txt.cfg custom/txt.cfg
*** orig/txt.cfg	2016-10-12 04:45:19.000000000 +0200
--- custom/txt.cfg	2017-02-12 00:34:01.977187336 +0100
***************
*** 1,7 ****
! default install
  label install
  	menu label ^Install
- 	menu default
  	kernel linux
  	append vga=788 initrd=initrd.gz --- quiet 
  label cli
--- 1,11 ----
! default unattended
! label unattended
! 	menu label ^Unattended Install
! 	menu default
! 	kernel linux
! 	append vga=788 initrd=initrd.gz auto=true priority=critical preseed/file=/preseed.cfg --- quiet
  label install
  	menu label ^Install
  	kernel linux
  	append vga=788 initrd=initrd.gz --- quiet 
  label cli
